inbm/dispatcher-agent/dispatcher/schedule/schedules.py

```diff
@@ -14,8 +14,9 @@ class Schedule:
     """ Represents a Base class for schedule objects """
     request_id: str
     schedule_id: Optional[int] = field(default=None)
-    # priority, schedule_id, job_id from DB
-    job_id: Optional[Tuple[int, int, int]] = field(default=None)
+    job_id: Optional[str] = field(default=None)
+    # priority, schedule_id and task_id from schedule_job table stored in DB
+    schedule_job_id: Optional[Tuple[int, int, int]] = field(default=None)
```

Comment:
this is unclear--maybe use a dataclass instead of a Tuple?

Reply:
Found a way to get rid of the tuple.